### DISCUSSION TITLE
Localhost

### DISCUSSION THREAD

**Discussion by EagL3-GIT** [mircokroon/minecraft-world-downloader#200] [Q&A]
How to connect to localhost? There is an error from minecraft when I try to connect localhost IP.

**Comment by LilBamaBeets** [comment]
Run Minecraft.
On the main menu, click Multiplayer.
Click Add Server.
Name your server in the Server Name field.
Type "localhost" in the Server Address field.
Click Done.
Select your server and click Join Server

**Reply by WhynotFulp** [reply]
Hey can you share screen shots of how you were able to do this?
